docs/sources/enterprise/request-security.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Request security"
+description = "Grafana Enterprise request security"
+keywords = ["grafana", "security", "enterprise"]
+weight = 110
++++
+
+# Request security
+
+> **Note:** Available in Grafana Enterprise v7.4+.
```

Comment:
```suggestion
> **Note:** Available in Grafana Enterprise v7.4 and later versions.
```

Reply:
@xlson, please apply this format throughout.